tokens/color/icon.yml

```
@@ -64,6 +64,26 @@ color:
           type: color
 
     status:
+      disabled:
+        _:
+          $value:
+            - '{color.gray.50}'
```

```suggestion
            - '{color.gray.40}'
```

`--rh-color-icon-status-disabled` on light color schemes has been changed to `gray-40` per @coreyvickery. https://github.com/RedHat-UX/red-hat-design-system/pull/2596 and https://github.com/RedHat-UX/red-hat-design-system/pull/2593 are related PRs.